tests/test_store.py

```diff
@@ -218,6 +218,32 @@ def test_getDatasetManifest(self, synapse_store, downloadFile):
             # return manifest id
             assert manifest_data == "syn51204513"
 
+    def test_synapse_rest_request(self, synapse_store):
```

Comment:
Using existing client functionality will allow you to remove this test.  That said, i think what's more important is to test `_update_table_uuid_column` this function.

Reply:
@thomasyu888 thanks for pointing that out! before this PR we weren't able to easily test `_update_table_uuid_column` due to the difficulty in triggering specific case where it's actually needed and the operations it was doing. It might be easier now that we've switched to using the rest calls